templates/shared/forms/interactive/storage.html

```diff
@@ -4,263 +4,42 @@
-              <input type="checkbox" aria-labelledby="consuming-storage-File-system-NAS" class="p-checkbox__input">
-              <span class="p-checkbox__label" id="consuming-storage-File-system-NAS">File system (NAS)</span>
-            </label>
-
-            <div class="js-other-container">
-              <label class="p-checkbox">
-                <input class="p-checkbox__input js-other-container__checkbox" type="checkbox" aria-labelledby="consuming-storage-other">
-                <span class="p-checkbox__label" id="consuming-storage-other">Other</span>
-              </label>
-              <input type="text" id="consuming-storage-other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other">
-            </div>
-          </div>
-        </div>
-      </div>
-      <div class="js-formfield">
-        <h3 class="p-heading--5">Are you currently paying for redundant data storage?</h3>
-        <div class="row u-no-padding">
-          <div class="col-4 u-sv3">
-            <label class="p-radio">
-              <input class="p-radio__input" type="radio" aria-labelledby="storage-Yes" name="redundant-storage" value="Yes">
-              <span class="p-radio__label" id="storage-Yes">Yes</span>
-            </label>
-            <label class="p-radio">
-              <input class="p-radio__input" type="radio" aria-labelledby="storage-No" name="redundant-storage" value="No">
-              <span class="p-radio__label" id="storage-No">No</span>
-            </label>
-            <label class="p-radio">
-              <input class="p-radio__input" type="radio" aria-labelledby="storage-Dont-know" name="redundant-storage" value="I don't know">
-              <span class="p-radio__label" id="storage-Dont-know">I don't know</span>
-            </label>
-          </div>
-        </div>
-      </div>
+    <div class="js-pagination js-pagination--1">
       <div class="row u-no-padding">
         <div class="u-sv3 js-formfield">
           <h3 class="p-heading--5">Tell us about your project and team</h3>
-          <textarea id="about-your-project-or-interest" name="sector-specific-requirements" aria-label="Tell us about your project and team" placeholder="Where are you located? Do you already have OpenStack? Is this initiative for a specific workload or for the business as a whole? Do you have particular regulatory constraints or priorities?" rows="3"></textarea>
+          <p>For example, where are you located? Is this for a specific workload or for the business as a whole? How much data are you planning on storing, and will there be ongoing growth? Do you know what level of performance you need? Do you have regulatory constraints or priorities?</p>
+          <textarea id="about-your-project-or-interest" name="sector-specific-requirements"
+            aria-label="Tell us about your project and team"
+            placeholder=""
+            rows="3"></textarea>
```

Comment:
Do you know if this `textarea` is placed outside the `form` element for a reason? I would expect it to be a child of it, in order to send its data when the form is submitted. For example, that's the case for the same kind of form that we have in [anbox-cloud.io](https://anbox-cloud.io/#get-in-touch) and [microcloud.is](https://microcloud.is/#get-in-touch).
I see that the `textarea` was outside the form also before, so this could be intentional, but it could also be a mistake since the beginning. Could you please check?

Reply:
It is intentionally outside of form, so that user can still see the suggested text when they are typing in the text field (better usability0